freqtrade/commands/cli_options.py

```diff
@@ -668,4 +668,10 @@ def __init__(self, *args, **kwargs):
         help='Specify additional lookup path for freqaimodels.',
         metavar='PATH',
     ),
+    "freqai_backtest_live_models": Arg(
+        '--freqai-backtest-live-models',
+        help='Run backtest with ready models.',
+        action='store_true',
+        default=False,
```

Comment:
Be careful with setting defaults here.

if you do this means you will not have the possibility to set this parameter via configuration.
A better approach to define defaults is in `constants.py` via validation (if necessary) - or by assuming it's non-mandatory and checking with `config.get(paramname, default)`.